File: src/main/java/com/deneb/client/features/huds/info/Ping.java
package com.deneb.client.features.huds.info;

import com.deneb.client.client.GuiManager;
import com.deneb.client.features.Category;
import com.deneb.client.features.HUDModule;
import com.deneb.client.utils.ChatUtil;
import com.deneb.client.utils.LagCompensator;

import java.awt.*;

@HUDModule.Info(name = "Ping", x = 170, y = 170, width = 100, height = 10,category = Category.INFO)
public class Ping extends HUDModule {

    @Override
    public void onRender() {

        int fontColor = new Color(GuiManager.getINSTANCE().getRed() / 255f, GuiManager.getINSTANCE().getGreen() / 255f, GuiManager.getINSTANCE().getBlue() / 255f, 1F).getRGB();

        int privatePingValue = LagCompensator.globalInfoPingValue();
        String Final = "Ping " + ChatUtil.SECTIONSIGN + "f" + privatePingValue;

        font.drawString(Final, this.x + 2, this.y + 4, fontColor);

        this.height = font.getHeight() * 2;
        this.width = font.getStringWidth(Final) + 4;

    }

}

File: src/main/java/club/deneb/client/features/huds/info/Ping.kt
package club.deneb.client.features.huds.info

import club.deneb.client.client.GuiManager.red
import club.deneb.client.client.GuiManager.green
import club.deneb.client.client.GuiManager.blue
import club.deneb.client.features.HUDModule
import club.deneb.client.features.Category
import club.deneb.client.utils.LagCompensator
import club.deneb.client.utils.ChatUtil
import java.awt.Color

@HUDModule.Info(name = "Ping", x = 170, y = 170, width = 100, height = 10, category = Category.INFO)
class Ping : HUDModule() {
    override fun onRender() {
        val fontColor = Color(red / 255f, green / 255f, blue / 255f, 1f).rgb
        val privatePingValue = LagCompensator.globalInfoPingValue()
        val final = "Ping " + ChatUtil.SECTIONSIGN + "f" + privatePingValue
        font.drawString(final, (x + 2).toFloat(), (y + 4).toFloat(), fontColor)
        height = font.height * 2
        width = font.getStringWidth(final) + 4
    }
}